swapi/src/app/app.module.ts

```diff
@@ -0,0 +1,41 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { SignInComponent } from './sign-in/sign-in.component';
+import { SignUpComponent } from './sign-up/sign-up.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
+import { MatFormFieldModule } from "@angular/material/form-field";
+import { MatIconModule } from "@angular/material/icon";
+import { MatDialogModule } from "@angular/material/dialog";
+import { MatButtonModule } from "@angular/material/button";
+import { MatInputModule } from "@angular/material/input";
+
+const materialModules = [
+  MatFormFieldModule,
+  MatIconModule,
+  MatDialogModule,
+  MatButtonModule,
+  MatInputModule
+];
+
+@NgModule({
+  declarations: [
+    AppComponent,
+    SignInComponent,
+    SignUpComponent
+  ],
+  imports: [
+    BrowserModule,
+    AppRoutingModule,
+    BrowserAnimationsModule,
+    ReactiveFormsModule,
+    ...materialModules,
+    FormsModule
+  ],
+  providers: [],
```

Comment:
this is meaningless, please remove

Reply:
Wtf are you talking about ??? You left empty providers field in code , which makes no sense at all